sysconfig/bbpv/tools/modules.yaml

```diff
@@ -76,6 +76,10 @@ modules:
       - '%clang'
       - '%pgi'
       - 'cmake'
+    likwid:
+      environment:
+        set:
+          KMP_INIT_AT_FORK: 'FALSE'
```

Comment:
Just out of curiosity.. 
https://github.com/RRZE-HPC/likwid/releases
If you look at the changelog of 4.3.2 to me it seems that this env variable is set by likwid itself... it doesn't hurt to set it in the module too.

Reply:
thanks for pointing that out. I put constraint ` likwid@:4.3.1` to restrict it up to 4.3.1.